tests/store/test_datasource.py

```diff
@@ -126,6 +126,8 @@ def test_versioning(capfd, bucket):
 
     min_keys = d.versions()
 
+    raise ValueError("Should this return data keys instead of datasource keys?")
```

Comment:
@gareth-j - An explicit raised `ValueError` is the cause of test errors - any idea why this was included and how this got into the code?